honeybee_radiance_command/rpict.py

```diff
@@ -0,0 +1,90 @@
+"""rpict command."""
+
+from .options.rpict import RpictOptions
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+import honeybee_radiance_command._typing as typing
+
+
+class Rpict(Command):
+    """rpict command."""
+
+    __slots__ = ('_octree', '_view')
+
+    def __init__(self, options=None, output=None, octree=None, view=None):
+        """Command.
+
+        Args:
+            options: Command options. It will be set to Radiance default values if not
+                provided by user.
+            output: File path to the output file (Default: None).
+            octree: File path to the octree file (Default: None).
+            view: File path to the octree file (Default: None).
+        """
+        Command.__init__(self, options, output)
+        self.options = options
+        self.output = output
+        self._octree = octree
+        self._view = view
+
+    @property
+    def options(self):
+        """Rpict options."""
+        if not self._options:
+            self._options = RpictOptions()
+        return self._options
+
+    @options.setter
+    def options(self, prop, value):
+        if value:
+            assert hasattr(self._options, prop), 'Expected RpictOptions'
```

Comment:
should we use isinstance here?

```suggestion
            assert isinstance(self._options, prop), 'Expected RpictOptions'
```

Reply:
Complied